### PR TITLE
feat: deployment readiness and liveness probes

### PR DESCRIPTION
**Pull request by f0rmiga**
Added readiness and liveness probes. Also, exposed some configuration to the `values.yaml`.

### CODE REVIEW

File: charts/minibroker/templates/service.yaml [hunk 1]
```diff
@@ -10,4 +10,4 @@ spec:
   ports:
   - protocol: TCP
     port: 80
-    targetPort: 8080
+    targetPort: {{ .Values.broker.deployment.port }}
```
Comment (mook-as) [inline charts/minibroker/templates/service.yaml:13]:
```suggestion
    targetPort: broker
```
I find pointing at the port _name_ to be generally clearer.

File: charts/minibroker/values.yaml [hunk 2]
```diff
@@ -11,6 +11,13 @@ tls:
   # base-64 encoded PEM data for the private key matching the certificate
   key:
 
+# The service broker server configuration.
+broker:
+  # The deployment configuration.
+  deployment:
+    # A port for the service broker HTTP API.
+    port: 8080
```
Comment (mook-as) [inline charts/minibroker/values.yaml:19]: I don't understand why this wants to be configurable.  It's only used in the container; everybody outside the helm chart should be using the service (which is still on port 80).

File: charts/minibroker/templates/service.yaml (hunk 1 above)
Comment (f0rmiga) [reply charts/minibroker/templates/service.yaml]: Fixed in https://github.com/kubernetes-sigs/minibroker/pull/81/commits/e2dbcfd44fc89d859186db6608d199a3d28f67ec.

File: charts/minibroker/values.yaml (hunk 2 above)
Comment (f0rmiga) [reply charts/minibroker/values.yaml]: Indeed. My fault. Fixed with https://github.com/kubernetes-sigs/minibroker/pull/81/commits/e2dbcfd44fc89d859186db6608d199a3d28f67ec.

File: charts/minibroker/values.yaml [hunk 3]
```diff
@@ -13,10 +13,10 @@ tls:
 
 # The service broker server configuration.
 broker:
-  # The deployment configuration.
-  deployment:
+  # The service configuration.
+  service:
     # A port for the service broker HTTP API.
-    port: 8080
+    port: 80
```
Comment (mook-as) [inline charts/minibroker/values.yaml:19]: Not sure we'd ever need to change it, but that's okay, at least there's some plausible use.

**Pull request merged**